src/Microsoft.ServiceFabric.Actors/Diagnostics/ActorMethodInfo.cs

```diff
@@ -0,0 +1,24 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System.Reflection;
+
+namespace Microsoft.ServiceFabric.Actors.Diagnostics
+{
+    internal class ActorMethodInfo
+    {
+        internal ActorMethodInfo() { }
```

Comment:
This constructor appears to be redundant because C# should provide an empty constructor by default.